### ISSUE TITLE
Schedule tasks in calendar

### ISSUE THREAD

**Issue by 13hannes11** [nextcloud/calendar#2619]
**Is your feature request related to a problem? Please describe.**
I manage many things I do via the task app of nextcloud. However, keeping track of my todos and calendar is a bit of a hastle. Also time planning on when to do which todos currently has to be done completely on my own.

**Describe the solution you'd like**
I would like to be able to schedule tasks as an appointment in free calendar slots. This would allow me to do easy planning for my week.

**Describe alternatives you've considered**
Currently the only alternative solution I see is the use of another service or to manually create events for everything.

**Additional context**
Inspiration could be taken from the calendar of any.do calendar.

**Comment by ursulean** [comment]
Agreed, a common workflow (including my own) is to keep unscheduled tasks until I find a day/time slot to schedule them into. This is currently done by switching back and forth from the Calendar and Tasks apps, and completely bypasses the convenient drag and drop scheduling capabilities of FullCalendar.

Since the tasks are being fetched anyway with 'Show tasks in calendar' checked, it would just be a matter of rendering them somewhere on the sidebar as a drop-down list for example, and allowing drag and drop to schedule them directly, which FullCalendar supports.

**Comment by st3iny** [comment]
@13hannes11 Hi, we just merged https://github.com/nextcloud/calendar/pull/7110 which seems to address this issue. Do you think this request to be done or are there some parts missing?

**Comment by 13hannes11** [comment]
Hi yes I'd say this fixes it.
